plugin/inc/ticket.class.php

```diff
@@ -515,6 +515,8 @@ public function showForm($ID, $options = []): bool
             if ($ID > 0 && ($lastClosedTicket->customfields->fields['effective_date_field'] ?? '') > $this->customfields->fields['effective_date_field']) {
                 $templateParams['total2BlackDisabled'] = true;
                 $templateParams['total2ColorDisabled'] = true;
+            } else {
+                $templateParams['countersDefaultValues'] = PluginIserviceTicket::getCountersDefaultValues($this->printer ?? new PluginIservicePrinter(), $this,  $lastClosedTicket) ?? [];
```

Comment:
`getCountersDefaultValues` will never return null (by declaration it always returns an array!)
```suggestion
                $templateParams['countersDefaultValues'] = PluginIserviceTicket::getCountersDefaultValues($this->printer ?? new PluginIservicePrinter(), $this,  $lastClosedTicket);
```